### ISSUE TITLE
Count Syllables 

### ISSUE THREAD

**Issue by mrigankdembla** [aayushi-droid/Edabit-Solutions#308]
Before jumping into Pr first comment for assign.

**Problem statement** : Create a function that counts the number of syllables a word has. Each syllable is separated with a dash -.
**Probelm Link** : https://edabit.com/challenge/zRe5fx3MbiBxcrvXH
**Choose one Cateorgy** - Easy
**question category** : Strings,Fundamentals

---
Before submitting a PR please Check some details.

- [x] fileName formet should be in camelcase
    example- returnTheNextNumber.py
- [x]  Choose label for Problem , example- "good first issue", "easy"
- [x]  In file there should be problem statement and link to problem
```cpp
/* Problem-Task : This program will add two numbers
 * Problem Link : https://edabit.com/challenge/SFzHtm63XT6EYNHWY
*/
int main() {
    int bla;
    return 0;
}
```

This repository is vaild for HacktoberFest2020

Read before making PR - [How to contribute on Github](https://www.dataschool.io/how-to-contribute-on-github/)
Good Luck,

**Comment by mrigankdembla** [comment]
Please assign this issue to me, thanks!

**Comment by mrigankdembla** [comment]
Resolved this, please review and merge PR #309
